how-to/use_terraform.rst

```diff
@@ -0,0 +1,133 @@
+JAAS: Using Terraform
+=====================
+
+Introduction
+------------
+
+In this how-to we will be showing you how to use Terraform with JAAS.
+
+Prerequisites
+-------------
+
+For this how-to you will need the following:
+
+- An identity provider that can be used to create OAuth2 client credentials
+- Client credentials (`client_id` and `client_secret`) generated by the above identity provider.
+- A deployed JIMM configured to trust the identity provider. For instructions on 
+  how to deploy JIMM read :doc:`deploy_jimm`.
+- A Juju 3.5 controller added to JIMM that can be used to control your chosen cloud. For instructions
+  on how to add one read :doc:`add_controller`.
+- A Juju 3.5 client
+- Cloud credentials for the chosen cloud.
+- Basic knowledge of Terraform, Juju Terraform provider and Juju.
+
+Registering client credentials
+------------------------------
+
+Before we can use client credentials generated by your chosen identity provider we need
+to register them.
+
+1. Install the JAAS snap:
+
+    ``sudo snap install jaas --channel latest/candidate``
+
+2. Register the cloud credential:
+
+    ``juju jaas add-service-account <client ID>``
```

Comment:
In fact you don't need the `jaas` part so just `juju add-service-account ...`

Reply:
done